src/index.js

```diff
@@ -0,0 +1,9 @@
+import './style.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+import loadEventListeners from './load';
+
+loadEventListeners();
```

Comment:
💡  It's best practice to end each file with an empty line, according to best practices.